tools/github/README.md

```diff
@@ -0,0 +1,51 @@
+# Overview
+
+This area is focused on the management of the Azure SDK GitHub repositories, containing tools and data used for that purpose.  
+
+## Prerequisites
+
+- **PowerShell:** In order to execute the scripts that perform the automation, PowerShell is needed.  The latest version of [PowerShell Core](https://github.com/PowerShell/PowerShell/blob/master/README.md) is recommended.
+
+- **GitHub CLI:** Repository data, such as milestones and labels, are read and written using the [GitHub CLI](https://github.com/cli/cli/tree/trunk#installation).  The latest version is recommended.
+
+## Structure
+
+* **root**  
+  _The root contains the core scripts for managing data and the set of centrally managed Azure SDK repositories._
+
+ ## Scripts
+ 
+ ### `Add-AzsdkMilestones.ps1`
+ 
+ _Creates a set of milestones in one or more repositories.  The milestones follow Azure SDK conventions for naming and due dates.  With the default parameter set, milestones will be created for the next 6 months in repositories in the 'repositories.txt' file._
+   
+ ```bash
```

Comment:
Why are you showing bash? Just run the scripts in `pwsh` and use `powershell` or `pwsh` (either should work and formats the same) for the code fence. Besides, many people may actually use `powershell.exe` - Windows PowerShell - instead of PowerShell (formerly "Core").